### PR TITLE
Securesign operator updates

### PR DESCRIPTION
**Pull request by sallyom**
remove unused securesign jobs & configs and add operator integration tests

### CODE REVIEW

File: ci-operator/config/securesign/secure-sign-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+- `securesign-operator`
+  - [Containerfile](https://github.com/securesign/secure-sign-operator/blob/main/Dockerfile)
+  - This Dockerfile is used to build a container image for the TAS Operator. It starts with a Golang base image and sets up the workspace. It then copies the Go Modules manifests and source code files into the container. The build process includes installing dependencies, setting build arguments, and compiling the Go code into a binary called "securesign-operator." The final image is based on Red Hat Universal Base Image (UBI) and includes additional packages and setup steps. The entry point of the container is set to execute the "jaeger-operator" binary.
+- `securesign-bundle`
+  - [Containerfile](https://github.com/securesign/secure-sign-operator/blob/main/bundle.Dockerfile)
+  - This Dockerfile is used to build the Jaeger Operator bundle. It creates an image from scratch, indicating that it starts with an empty base. The labels define metadata for the bundle, including its media type, manifests, metadata, package, channels, and metrics. The COPY commands copy the bundle's manifests and metadata files to the appropriate locations as specified by the labels.
```
Comment (cooktheryan) [inline ci-operator/config/securesign/secure-sign-operator/README.md:51]: just patch this up later. Bad copy paste

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

File: ci-operator/config/securesign/secure-sign-operator/README.md (hunk 1 above)
Comment (sallyom) [reply ci-operator/config/securesign/secure-sign-operator/README.md]: done!

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse periodic-ci-securesign-secure-sign-operator-main-security

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by sallyom** [comment]
@cooktheryan this should be ready to merge - ensures the operator bundle and images are built and operator is installed successfully.

**Pull request merged**